app/views/account/robloxButton.vue

```diff
@@ -0,0 +1,159 @@
+<script>
+import VueConfirmDialog from 'vue-confirm-dialog'
+const OAuth2Identities = require('collections/OAuth2Identities')
+
+Vue.use(VueConfirmDialog)
+Vue.component('vue-confirm-dialog', VueConfirmDialog.default)
+
+export default Vue.extend({
```

Comment:
we don't need to add vue.extend

Reply:
It seems we need when including to AccountSettings, which is Backbone.